CMSgov/cms-carts-seds
# MDCT-296: Functionality and Seeding for FMAP and ACS Tables

# Description

Closes [MDCT-296](https://qmacbis.atlassian.net/browse/MDCT-296). The `/state` endpoint should return FMAP and ACS data from the DB to load and populate Section 2 (Part 2) and Section 5 (Table 3) of CARTS reports.

## How to test

* Run locally `./dev local`
* Sign in to CARTS
* Open up a state report and navigate to **Section 2**
* Ensure the table in **Part 2: Number of Uninsured Children in Your State** is populated with either data or 'Not Available' (there should be no empty rows)
* Navigate to **Section 5**
* Ensure the _eFMAP_ row in **Table 3: Federal and State Shares** is populated with either data or 'Not Available' (there should be no empty rows)

Enjoy! 🪄 

## Dependencies

N/A

# Get it done

Cool things.

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
